src/site/_includes/components/CodePattern.js

```diff
@@ -16,18 +20,18 @@ module.exports = (patternId, height = 300) => {
         Prism.languages[type],
         type,
       );
-      return html`
+      return `
         <web-tab title="${asset.type}" data-label="${asset.type}">
           <pre><code class="language-${asset.type}">${content}</code></pre>
         </web-tab>
       `;
     })
     .join('\n');
 
-  return html`<div class="code-pattern flow flow-space-400">
+  return `<div class="code-pattern flow flow-space-400">
```

Comment:
why drop the html common-tags thing?
(To be clear, it seems to just - strip whitespace? Maybe we should use `safeHtml`?)
I don't need you to solve this now. Just wondering why.

Reply:
Because they were removing line breaks and we need to preserve line breaks to show code as multiline strings rather than one big blob.